howitworks.md

```diff
@@ -6,6 +6,18 @@ top_graphic: 3
 redirect_from: "/howitworks"
 ---
 
+# What is Let's Encrypt?
+
+Let's Encrypt allows you to secure your website by concealing the data
+sent and received between your website and the users of your website from prying
+eyes. Prior to the creation of Let's Encrypt, securing your website with SSL/HTTPS
+was often costly and a hassle. Basic security measures shouldn't be a hassle, nor 
+should they be prohibitively expensive in an era where theft of data is becoming
+increasingly common. Let's Encrypt was created for and is here to solve that issue,
```

Comment:
I like this last sentence, but it's a bit confusing since we haven't yet defined "certificate" and how it relates to HTTPS. Since we define that in the next section, let's just drop this last sentence.